src/sugar/orderbook.ts

```diff
@@ -74,21 +72,21 @@ async function getOrderbook(
     (reverseOfferResult) => reverseOfferResult.result.offers,
   )
 
-  // Sort the orders
-  // for both buys and sells, lowest quality is closest to mid-market
-  // we sort the orders so that earlier orders are closer to mid-market
-
   const orders = [...directOffers, ...reverseOffers]
-  // separate out the orders amongst buy and sell
+  // separate out the buy and sell orders
   const buy: BookOffer[] = []
   const sell: BookOffer[] = []
   orders.forEach((order) => {
-    if ((order.Flags & OfferLedgerFlags.lsfSell) !== 0) {
+    // eslint-disable-next-line no-bitwise -- Bitwise AND
+    if (order.Flags & OfferLedgerFlags.lsfSell) {
```

Comment:
It's good practice to explicitly state the condition
```suggestion
    if (order.Flags & OfferLedgerFlags.lsfSell !== 0) {
```

Reply:
Linter finds it unnecessary!

Reply:
As in the linter has a warning when you add that?

Reply:
Yes